src/classes/effect_init.py

```diff
@@ -25,7 +25,8 @@
  You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
  """
-
+from classes.info import USER_PATH
```

Comment:
You might add a new YOLO_PATH to info.py, so it gets auto created on launch... and thus, the user will already have an empty ~/.openshot_qt/yolo/ folder existing.